packages/loaders/stories/2-Dots.stories.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Story, Meta } from '@storybook/react';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { Dots, IDotsProps } from '@zendeskgarden/react-loaders';
+
+export default {
+  title: 'Components/Loaders/Dots',
+  component: Dots
+} as Meta;
+
+const StyledExampleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+interface IDefaultStoryProps extends IDotsProps {
+  size: number;
+}
+
+export const Default: Story<IDefaultStoryProps> = ({ color, delayMS, duration, size }) => {
+  return (
+    <StyledExampleWrapper>
+      <Dots color={color} delayMS={delayMS} duration={duration} size={`${size}px`} />
+    </StyledExampleWrapper>
+  );
+};
+
+Default.argTypes = {
+  size: {
+    name: 'size (px)',
+    control: { type: 'range', min: 30, max: 250, step: 1 }
+  },
+  duration: {
+    name: 'duration (ms)',
```

Comment:
[nit] the parenthetical "(px)" and "(ms)" are probably not necessary as the units are clarified in the prop descriptions.

Reply:
I've removed these here as well in the `Spinner` docs